zinc/src/main/java/xsbti/compile/ZincCompilerUtil.java

```diff
@@ -8,12 +8,14 @@
 package xsbti.compile;
 
 import java.io.File;
+import xsbti.F0;
+import xsbti.Logger;
 
 /**
  * Defines a util interface to get Scala compilers and the default implementation
  * of the Scala incremental compiler if only {@link IncrementalCompiler} is required.
  */
-public interface ZincCompilerUtil {
+public class ZincCompilerUtil {
```

Comment:
Can you add a comment to the Scaladoc that explains the reason why this is turned into a class instead of being an interface and linking to the issue?